### PR TITLE
include flag

### PR DESCRIPTION
**Pull request by lsbardel**

### CODE REVIEW

File: openapi/data/db.py [hunk 1]
```diff
@@ -13,18 +13,23 @@
...
+def dataclass_from_table(
+        name, table, *, exclude=None, include=None, required=None):
+    """Create a dataclass from an sqlalchemy table
+    """
     columns = []
     exclude = set(exclude or ())
     for col in table.columns:
         if col.name in exclude:
             continue
+        if include and col.name not in include:
```
[the remark this answers is not in the document]
Comment (lsbardel) [reply openapi/data/db.py]: not sure what you mean here

File: openapi/data/db.py [hunk 2]
```diff
@@ -39,72 +44,73 @@ def _(f):
...
     )
 
 
-def info(col):
+def info(col, required):
+
     data = dict(
         description=col.doc,
-        required=not col.nullable
+        required=not col.nullable if required is not False else False
```
[the remark this answers is not in the document]
Comment (lsbardel) [reply openapi/data/db.py]: I don't follow, `required=False` overrides the nullable property. I don't want to do that by default

File: openapi/data/db.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ConorStevenson) [reply openapi/data/db.py]:
```
def dataclass_from_table(
        name, table, *, exclude=(), include=(), required=None):
    for col in table.columns:
        if col.name in exclude or col.name not in include:
            continue
```

[the remark this answers is not in the document]
Comment (ConorStevenson) [reply openapi/data/db.py]: Or if you still want superfast sets
```
cols = set(table.columns) & set(include) - set(exclude)
```

**Pull request merged**